demo/src/main/java/eu/kevin/demo/main/MainFragment.kt

```diff
@@ -104,11 +104,15 @@ class MainFragment : Fragment(), MainViewCallback {
     }
 
     override fun onMakeBankPaymentPressed() {
-        viewModel.initializePayment(PaymentType.BANK)
+        viewModel.initializePayment(PaymentType.BANK, isHybrid = false)
```

Comment:
if payment type is BANK, isHybrid won't affect anything, so this part should be rethought